tests/ops/op_math/test_adjoint.py

```diff
@@ -884,6 +884,21 @@ def test_single_observable(self):
         qs = qml.tape.QuantumScript.from_queue(q)
         assert len(qs) == 0
 
+    def test_correct_queued_operators(self):
+        """Test that args and kwargs do not add operators to the queue."""
+
+        dev = qml.device("default.qubit")
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.adjoint(qml.QSVT)(qml.X(1), [qml.Z(1)])
+            qml.adjoint(qml.QSVT(qml.X(1), [qml.Z(1)]))
+            return qml.state()
+
+        circuit()
+        for op in circuit.tape.operations:
+            assert op.name == "Adjoint(QSVT)"
```

Comment:
```suggestion
        
        with qml.queuing.AnnotatedQueue() as q:
            qml.adjoint(qml.QSVT)(qml.X(1), [qml.Z(1)])
            qml.adjoint(qml.QSVT(qml.X(1), [qml.Z(1)]))

        for op in q.queue:
            assert op.name == "Adjoint(QSVT)"
```
We should avoid involving the qnode or the device in unit tests for basic queuing functionality.

Reply:
Oh good to know 👌
Changed in all tests